packages/main/src/Avatar.hbs

```diff
@@ -17,4 +17,8 @@
 	{{else if initials}}
 		<span class="ui5-avatar-initials">{{validInitials}}</span>
 	{{/if}}
+
+	{{#if showBadge}}
+		{{Badge}}
```

Comment:
This works with `lit-html`, but makes the template less declarative overall. Of course, it's still better than adding the badge imperatively with DOM APIs afterwards. It's a consequence of the solution as a whole (compared to slots, which is in general a more explicit and declarative solution)